migration/sql-files/029-identities-foreign-key.sql

```diff
@@ -0,0 +1,2 @@
+-- Refactor Identities: add a foreign key constraint
+alter table identities add constraint identies_user_id_users_id_foreign foreign key (user_id) REFERENCES users (id);
```

Comment:
@kwk @xcoulon this reasoning can be applied to other tables such as spaces, iterations, etc... Are we gonna apply `ON DELETE RESTRICT` to all ? it would be good to have this clarified